docs/serving/samples/knative-routing-go/README.md

```diff
@@ -21,6 +21,8 @@ the Login service.
    - In this example, we use `example.com`. If you don't have a domain name, you
      can modify your hosts file (on Mac or Linux) to map `example.com` to your
      cluster's ingress IP.
+   - If you have a domain name for your Kubernetes cluster, please stay with it.
```

Comment:
I would say "If you have configured a custom domain for your Knative installation, we will refer to it as <your domain name> in the rest of this document"

Reply:
Done. thx.